readme.md

```diff
@@ -288,6 +288,7 @@
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis)
 - [Information Retrieval](https://github.com/harpribot/awesome-information-retrieval) - Learn to develop your own search engine.
+- [Quantum Computing](https://github.com/desireevl/awesome-quantum-computing)
```

Comment:
> The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**